### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by kont-noor**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
```
Comment (DevMike) [inline app/controllers/users_controller.rb]: why do we need user controller when we have registrations controller?

File: app/views/users/show.html.haml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+%h1= @user.full_name
+= image_tag @user.avatar.url(:medium)
+%dl
+  %dt ICQ
+  %dd= @user.icq
+  %dt Skype
+  %dd= @user.skype
+  %dt Phone
```
Comment (DevMike) [inline app/views/users/show.html.haml]: should be localized(see attribute translations in yml)

File: Gemfile [hunk 3]
```diff
@@ -58,6 +58,9 @@ gem "money-rails", "~> 0.7.1"
 #markup editor
 gem "ckeditor", :git => "https://github.com/ginter/ckeditor.git"
 
+# attachments
+gem "paperclip"
```
Comment (DevMike) [inline Gemfile:62]: we use carrierwave for images in the project. Why do you use the paperclip?

File: db/migrate/20130102113506_add_names_contacts_and_avatar_to_user.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
+class AddNamesContactsAndAvatarToUser < ActiveRecord::Migration
```
Comment (DevMike) [inline db/migrate/20130102113506_add_names_contacts_and_avatar_to_user.rb:1]: 1) what do you remove and add same columns here?
2) self maybe skipped

File: spec/controllers/users_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe UsersController do
```
Comment (DevMike) [inline spec/controllers/users_controller_spec.rb]: these tests are worthless

File: app/views/users/show.html.haml [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+  %dd= @user.phone
+
+- if @user == current_user
```
Comment (DevMike) [inline app/views/users/show.html.haml]: are you sure that devise don't do it? If yes, please use cancan to check permissions

File: Gemfile (hunk 3 above)
Comment (kont-noor) [inline Gemfile:62]: decided to leave as is

File: db/migrate/20130102113506_add_names_contacts_and_avatar_to_user.rb (hunk 4 above)
Comment (kont-noor) [inline db/migrate/20130102113506_add_names_contacts_and_avatar_to_user.rb:1]: done

File: app/views/users/show.html.haml (hunk 2 above)
Comment (kont-noor) [inline app/views/users/show.html.haml]: done

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (kont-noor) [inline app/controllers/users_controller.rb]: leaved Users controller but remade it

File: spec/controllers/users_controller_spec.rb (hunk 5 above)
Comment (kont-noor) [inline spec/controllers/users_controller_spec.rb]: removed

File: app/views/users/show.html.haml (hunk 6 above)
Comment (kont-noor) [inline app/views/users/show.html.haml]: done with cancan

**Pull request merged**